src/test/java/org/folio/circulation/rules/Text2DroolsTest.java

```diff
@@ -432,18 +461,41 @@ public void shelvingLocation() {
       createLocation(FIRST_INSTITUTION_ID, FIRST_LIBRARY_ID, FIRST_CAMPUS_ID)),   is("policy-e"));
   }
 
+  @Test
+  public void overdueFinePolicy() {
```

Comment:
`overdueFinePolicy()` has been inserted between the existing methods `shelvingLocation()`and `shelvingLocationDefaultPriority()`. This is odd, can you move it either before or after them?